README.md

```diff
@@ -231,19 +231,19 @@ For more details about `CNAME`, read the official documentation: [Managing a cus
     cname: github.com
 ```
 
-### ⭐️ Disable `.nojekyll`
+### ⭐️ Enable Jekyll
 
-By default, this action adds the `.nojekyll` file to only the `master` and `gh-pages` branches. When the file already exists, this action does nothing.
+If you are using the static site generator Jekyll, set `enable_jekyll` to true.
```

Comment:
```suggestion
If you are using the GitHub Pages built-in static site generator Jekyll, set `enable_jekyll` to true.
```

Users who use Jekyll on Docker do not need to set `enable_jekyll` to true.

Reply:
Sorry, do not care. Jekyll seems not to read files which name starts with an underscore no matter it is built-in, or not.

Reply:
Tried to make it clearer now in the description.